src/main/kotlin/com/terraformation/backend/tracking/model/ObservationResultsModel.kt

```diff
@@ -117,33 +118,57 @@ data class ObservationMonitoringPlotResultsModel(
     val totalSpecies: Int,
 )
 
+interface BaseMonitoringResult {
```

Comment:
Why do we need this interface? Doesn't look like anything is taking it as a parameter or using it as a type.

Maybe it's to avoid repeating the KDoc comments for similar fields? In that case, the comments here should be updated to not refer specifically to zones any more.